static/app/components/eventTitleTreeLabel.tsx

```diff
@@ -22,15 +22,15 @@ function EventTitleTreeLabel({treeLabel}: Props) {
           if (index !== firstFourParts.length - 1) {
             return (
               <Fragment key={index}>
-                <PriorityPart highlight={highlight}>{label}</PriorityPart>
```

Comment:
this is an unrelated change entirely, right? can you describe what you're fixing here (wrt the css changes, not this line specifically)

Reply:
it appears to be a bug in max grouping level where the last elements linewrap/overflow if there are too many treelabelparts

ref https://getsentry.atlassian.net/browse/INGEST-172

Reply:
I removed this bug fix from this PR ;-)